docs/labs/installer.rst

```diff
@@ -264,7 +264,7 @@ Further reading
 
 Now that you've installed your own Flocker cluster, you may want to learn more about Flocker:
 
-* :ref:`Using Flocker <using>` (note that the ``flocker-deploy`` tool is installed on your system as ``uft-flocker-deploy``)
+* :ref:`Controlling Flocker <controlling-flocker>` (note that the ``flocker-deploy`` tool is installed on your system as ``uft-flocker-deploy``)
```

Comment:
:question: ``` :ref:`controlling-flocker` ```will render the same as ``` :ref:`Controlling Flocker <controlling-flocker>` ``` (it uses the title of the reference), and it won't risk being weird when the section title changes and this link title does not.